aesara/link/jax/dispatch.py

```diff
@@ -1047,3 +1048,48 @@ def random_variable(rng, size, dtype_num, *args):
         return (rng, smpl_value)
 
     return random_variable
+
+
+@jax_funcify.register(Erf)
+def jax_funcify_Erf(op, **kwargs):
+    def erf(x):
+        return jax.scipy.special.erf(x)
+
+    return erf
+
+
+@jax_funcify.register(Erfc)
+def jax_funcify_Erfc(op, **kwargs):
+    def erfc(x):
+        return jax.scipy.special.erfc(x)
+
+    return erfc
+
+
+# Commented out because jax.scipy does not have erfcx,
```

Comment:
Should we use the naive operations in the meanwhile? https://www.mathworks.com/help/matlab/ref/erfcx.html

Reply:
I dug around on the internet; the naive implementation can work but we'll probably very frequently run into underflow and overflow issues. (The matlab reference says so too.) 

I also looked on the JAX issue tracker. [Issue #1987](https://github.com/google/jax/issues/1987) is still open, but they're looking for (1) a license-compatible implementation (see [here](https://github.com/google/jax/pull/7372)) that is (2) not the naive version (see [here](https://github.com/google/jax/pull/3856)). 

In theory, we could copy the code from [Stack Overflow](https://stackoverflow.com/questions/39777360/accurate-computation-of-scaled-complementary-error-function-erfcx/39777361#39777361) but I'm not sure if it'll be license-compatible for us either.

Reply:
I think the SO license is fine for us, we just have to add a link in comment or something like that. But we can ask around not be sure

Reply:
Intuitively, this feels like an issue to raise and should be excluded out of this PR, as the discussion might creep ever larger.

Reply:
Definitely. We don't need to decide on this for the PR.

I am not sure about leaving the commented blocks in though